src/js/components/RangeSelector/RangeSelector.js

```diff
@@ -173,6 +179,20 @@ const RangeSelector = forwardRef(
       [onMouseMove],
     );
 
+    // keep the text values size consistent
+    useLayoutEffect(() => {
+      if (maxRef.current && minRef.current) {
+        maxRef.current.style.width = '';
+        minRef.current.style.width = '';
+        const width = Math.max(
+          maxRef.current.getBoundingClientRect().width,
+          minRef.current.getBoundingClientRect().width,
+        );
+        maxRef.current.style.width = `${width}px`;
+        minRef.current.style.width = `${width}px`;
```

Comment:
Not sure if it's related to this line, but the text is jumping side to side a little when I reach the "max" vs not:

https://user-images.githubusercontent.com/12522275/210617897-124cf17c-8b54-4a31-b91e-58c783403c5c.mov

Reply:
This should be improved with the latest changes.